extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -100,11 +103,56 @@ public static Builder repositoryAdapter() {
       final RepositoryPresenter<Object> untypedPresenter =
           (RepositoryPresenter<Object>) checkNotNull(presenter);
       presenters.add(untypedPresenter);
+      isDynamicPresenter.add(true);
       observables.add(untypedRepository);
       return this;
     }
 
     /**
+     * Specifies that the {@link RepositoryAdapter} being built should present the given
+     * {@code item} next (after all previously added item), using the given
```

Comment:
after all previously added repositories, items and static layouts (you'll have to say this whole list everywhere, lol)